dispatch/static/manager/src/js/components/fields/IntField.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import { TextInput } from '../inputs'
+
+export default class IntField extends React.Component {
+  onChange(val) {
+    val = val.replace(/[^\d]/, '')
+    this.props.onChange(Math.floor(val))
+  }
+
+  render() {
+    return (
+      <TextInput
+        placeholder={this.props.label}
+        value={this.props.data || ''}
+        fill={true}
+        type="text"
```

Comment:
type="number" doesn't neatly support integers. 

ie. it allows decimal places, and doesn't call onChange when they are inserted